pyvista/core/objects.py

```diff
@@ -469,10 +607,29 @@ def to_array(self):
         return np.flip(image.active_scalars.reshape(shape, order='F'), axis=1).swapaxes(1, 0)
 
     def plot(self, *args, **kwargs):
-        """Plot the texture as image data by itself."""
+        """Plot the texture as image data by itself.
+
+        If the texture is a cubemap, it will be displayed as a skybox.
+
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>> texture = examples.download_masonry_texture()
+        >>> texture.plot()
+
+        """
+        if self.cube_map:
+            return self._plot_skybox(*args, **kwargs)
         kwargs.setdefault("rgba", True)
         return self.to_image().plot(*args, **kwargs)
 
+    def _plot_skybox(self, *args, **kwargs):
+        """Plot this texture as a skybox."""
+        pl = pyvista.Plotter(**kwargs)
```

Comment:
Are we OK with silently ignoring any superfluous `*args` being passed?

Reply:
That's a mistake on my part.

```suggestion
    def _plot_skybox(self, *args, **kwargs):
        """Plot this texture as a skybox."""
        pl = pyvista.Plotter(*args, **kwargs)
```